doc/design.md

```diff
@@ -398,7 +398,7 @@ Needs: impl
 #### Retrieve GitHub Release Header from Release Letter
 `dsn~retrieve-github-release-header-from-release-letter~1`
```

Comment:
You need to update the revision number of the requirement if you change the semantics.